### PR TITLE
NOISSUE - Add latency metrics to dashboard

### PR DESCRIPTION
**Pull request by SammyOina**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](https://github.com/mainflux/mainflux/blob/master/CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Bumps grafana version and add latency metrics to dashboard

### Which issue(s) does this PR fix/relate to?
Relates to #1772 

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### CODE REVIEW

File: docker/addons/prometheus/grafana/example-dashboard.json [hunk 1]
```diff
@@ -3,7 +3,10 @@
     "list": [
       {
```
Comment (rodneyosodo) [inline docker/addons/prometheus/grafana/example-dashboard.json:4]: Can we add request information to avoid confusion

image

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Comment by arvindh123** [comment]
LGTM

**Review by arvindh123 — DISMISSED** [review]

File: docker/addons/prometheus/grafana/example-dashboard.json (hunk 1 above)
Comment (SammyOina) [reply docker/addons/prometheus/grafana/example-dashboard.json:4]: had both instant and range values, instant now not used

Comment (rodneyosodo) [reply docker/addons/prometheus/grafana/example-dashboard.json:4]: For instance we have 2 0.5s. They have different meanings which aren't displayed on the legend

File: docker/addons/prometheus/grafana/example-dashboard.json [hunk 2]
```diff
@@ -3,7 +3,10 @@
     "list": [
```
Comment (rodneyosodo) [inline docker/addons/prometheus/grafana/example-dashboard.json:3]: Let's not aggregate quantiles

**Review by drasko — DISMISSED** [review]
LGTM

**Comment by rodneyosodo** [comment]
You can use a query like this to replace the values with more readable format

```
label_replace(label_replace(label_replace(things_api_request_latency_microseconds, "quantile", "50th percentile", "quantile", "0.5"), "quantile", "90th percentile", "quantile", "0.9"), "quantile", "99th percentile", "quantile", "0.99")
```

Then on the legend you have it like

```
{{method}} - {{quantile}}
```

This will produce something like this

image

**Comment by rodneyosodo** [comment]
Rebase with master so that this changes are reflected https://github.com/mainflux/mainflux/commit/3b84258a0ce034b615be0eb63bee297439ecce40

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Add units to the graphs to be something like this 
image

**Review by rodneyosodo — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**